vigranumpy/test/test_segmentation.py

```diff
@@ -33,6 +33,49 @@ def test_labelMultiArray():
     _impl_test_labelMultiArray(numpy.uint8)
     _impl_test_labelMultiArray(numpy.uint32)
     _impl_test_labelMultiArray(numpy.float32)
+
+
+def _impl_test_applyMapping(dtype):
+    original = numpy.arange(100, dtype=dtype ).reshape(10,10)
+    mapping = { int(k) : int(k + 100) for k in original.flat[:] }
+
```

Comment:
BTW, Here, I'm converting from `np.uint8` (or whatever) to the standard Python `int` type to avoid errors from `boost::python::extract<>()` such as the following:

```
TypeError: No registered converter was able to produce a C++ rvalue of type unsigned char from this Python object of type numpy.uint8
```

Is there a clever way to teach `extract<>()` how to deal with `np.uint8`, etc.?  At the moment, users of `applyMapping()` will be required to convert their dict entries into `int` themselves, as I have done in this test.